user/parlib/signal.c

```diff
@@ -74,7 +74,10 @@ static void default_term_handler(int signr, siginfo_t *info, void *ctx)
 static void default_core_handler(int signr, siginfo_t *info, void *ctx)
 {
 	fprintf(stderr, "Segmentation Fault (sorry, no core dump yet)\n");
-	print_user_context((struct user_context*)ctx);
+	if (ctx)
+		print_user_context((struct user_context*)ctx);
+	else
+		fprintf(stderr, "No ctx for %s\n", __FUNCTION__);
```

Comment:
Really?  I feel like we should restrict sys_notify from sending signals that generate SIGSEGV and friends?